### DISCUSSION TITLE
CPP dbg tool

### DISCUSSION THREAD

**Discussion by ronen77** [microsoft/vscode-cpptools#8918] [Q&A]
Hi,

my vscode application is running on windows machine, connected via ssh to remote linux x86.
1. Do I need to download the win32 or linux x86 vsix file?
2. I have tried both, but i get an error message saying that the extension is not compatible with vscode 1.57.1
    that I am using (I have no intent of changing the 1.57.1 version)

Appreciate your comment, on which version of cpp dbg extension to download, and where from?

**Comment by michelleangela** [comment]
Starting with version 1.8.0 of the C++ extension, it requires VS Code version 1.63+.

If you would like to use VS Code version 1.57.1, you can try [version 1.5.1 of the C++ extension](https://github.com/microsoft/vscode-cpptools/releases/tag/1.5.1) which only requires VS Code 1.53+.

To run the extension on a remote Linux x86 machine, you would need to download `cpptools-linux32.vsix` and install that vsix on the remote machine. However, this Linux 32-bit version of the extension is only available up to 0.27.0 version (see https://github.com/microsoft/vscode-cpptools/issues/5346).

**Comment by ronen77** [comment]
Hi,

Following your suggestion, i will upgrade the entire system to use 1.63.2.
Where can I download vscode version 1.63.2 for windows as well as Linux
(debian)?

Regards,
Ronen Levy


‫בתאריך יום ו׳, 25 בפבר׳ 2022 ב-23:06 מאת ‪Michelle Matias‬‏ <‪
***@***.***‬‏>:‬

**Reply by michelleangela** [reply]
The latest VS Code version is available at https://code.visualstudio.com/Download. Previous VS Code versions are available at https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions.

The C/C++ Extension VSIX packages can be downloaded at https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools. The extension can also be downloaded/installed from the VS Code client via the "Extensions" panel menu.
